### PR TITLE
docs: clarify on `request.body` content and usage

### LINKED ISSUE #3389: Improve getting started documentation to include req.body
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the feature has not already been requested

### 🚀 Feature Proposal

Improved our documentation so it’s immediate for newcomers that the case described in https://github.com/fastify/fastify/discussions/3388 is not needed at all.

### Motivation

_No response_

### Example

_No response_

### PR DESCRIPTION
**Pull request by Fdawgs**
Closes #3389

#### Checklist

- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by jsumners — APPROVED** [review]
Looks good to me.

**Review by RafaelGSS — APPROVED** [review]
LGTM.

File: docs/Getting-Started.md [hunk 1]
```diff
@@ -363,6 +363,20 @@ fastify.get('/', opts, async (request, reply) => {
 By specifying a schema as shown, you can speed up serialization by a factor of 2-3. This also helps to protect against leakage of potentially sensitive data, since Fastify will serialize only the data present in the response schema.
 Read [Validation and Serialization](Validation-and-Serialization.md) to learn more.
 
+<a name="request-payload"></a>
+### Parsing request payloads
+Fastify parses `'application/json'` and `'text/plain'` request payloads natively, with the result accessible from the [Fastify request](Request.md) object at `request.body`.<br>
+The following example returns the parsed body of a request back to the client:
+
+```js
+const opts = {
```
Comment (climba03003) [inline docs/Getting-Started.md:372]:
```suggestion
const opts = {}
```

**Review by climba03003 — CHANGES_REQUESTED** [review]
Missing a closing bracket

**Review by climba03003 — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**